validator/sawtooth_validator/execution/executor.py

```diff
@@ -39,6 +40,10 @@
 LOGGER = logging.getLogger(__name__)
 
 
+TpSetting = namedtuple('TpSetting',
```

Comment:
Is this just used for storing one setting?  If so, it should probably be named for that particular setting

Reply:
Okay, I was following the format of `self._tp_settings_key = "sawtooth.validator.transaction_families`. Should it be called TransactionFamilies?

Reply:
That sounds like a good name